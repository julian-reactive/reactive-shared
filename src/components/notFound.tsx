import React from 'react'

import Box from '@mui/material/Box'
import Grid from '@mui/material/Grid'
import Paper from '@mui/material/Paper'

const NotFoundComponent: React.FC = () => {
  return (
    <Grid container>
      <Grid item xs={12}>
        <Paper>
          <Box p={5}>
            <h1 className='display-3'>404!</h1>
            <p className='lead'>Page not found</p>
          </Box>
        </Paper>
      </Grid>
    </Grid>
  )
}

export const NotFound = React.memo(NotFoundComponent)
