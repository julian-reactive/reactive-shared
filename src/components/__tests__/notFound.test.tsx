import React from 'react'

import { render } from '@testing-library/react'
import { NotFound } from '../notFound'

const getRendered = () => render(<NotFound />)

describe('notFound', () => {
  it('should match snapshot', () => {
    const { container } = getRendered()
    expect(container.firstChild).toMatchSnapshot()
  })
})