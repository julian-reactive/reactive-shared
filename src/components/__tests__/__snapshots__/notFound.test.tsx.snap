// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notFound should match snapshot 1`] = `
<div
  class="MuiGrid-root MuiGrid-container css-11lq3yg-MuiGrid-root"
>
  <div
    class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 css-1idn90j-MuiGrid-root"
  >
    <div
      class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
    >
      <div
        class="MuiBox-root css-1mb7ed4"
      >
        <h1
          class="display-3"
        >
          404!
        </h1>
        <p
          class="lead"
        >
          Page not found
        </p>
      </div>
    </div>
  </div>
</div>
`;
